patches/chrome-android-java-src-org-chromium-chrome-browser-notifications-NotificationPlatformBridge.java.patch

```diff
@@ -29,12 +29,3 @@ index 86ed1463f11a0caf0c549784591b1cc8125c392f..0c3ac7b9e689c08a6c402e6e1cc0bc3f
              @NotificationType int notificationType, String origin, String scopeUrl,
              String profileId, boolean incognito, boolean vibrateEnabled, String title, String body,
              Bitmap image, Bitmap icon, Bitmap badge, int[] vibrationPattern, long timestamp,
-@@ -699,7 +699,7 @@ public class NotificationPlatformBridge {
-                         notificationId /* notificationTag */, PLATFORM_ID /* notificationId */));
-     }
- 
--    private NotificationBuilderBase createNotificationBuilder(Context context, boolean hasImage) {
```

Comment:
private -> protected change can be done with asm

Reply:
@samartnik Mind taking a look at this one?

Reply:
`createNotificationBuilder` was removed along with this patch. We currently deal with notifications differently https://github.com/brave/brave-core/commit/c6e42169d0a386b1e26f4b73605d40aadb29039a

Reply:
ok, I didn't see the patch removed in the commits after this, but if it was then great